packages/frontend/src/components/common/DataExplorer/coordinates.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This is great - would it be possible to include these in `src/portal_configs.json` instead and read the values from there? That way we don't have two different files of constants for each city.